device/WakeOnLan/controller.js

```diff
@@ -0,0 +1,130 @@
+﻿'use strict';
+
+const BluePromise = require('bluebird');
+const fs = require('fs')
+const os = require('os');
+const net = require('net');
+const dgram = require('dgram');
+
+const PORT_NBR = 9;
+const DEFAULT_IPADDRESS = '255.255.255.255';
+
+let wolMap = "wol.map";
```

Comment:
Although I'd prefer JSON myself, I've dealt with enough users to know that non-programmers hate json/xml formats and to keep setup/config type files as simple as possible (like a simple x=y type of lines).  Since a majority will try this stuff without knowing how to program - I chose that format for ease.  If you feel strongly that it should be json - I'll switch to json (pretty easy)